nemo_text_processing/text_normalization/en/taggers/roman.py

```diff
@@ -40,28 +39,64 @@ class RomanFst(GraphFst):
     def __init__(self, deterministic: bool = True):
         super().__init__(name="roman", kind="classify", deterministic=deterministic)
 
-        def _load_roman(file: str):
-            roman = load_labels(get_abs_path(file))
-            roman_numerals = [(x, y) for x, y in roman] + [(x.upper(), y) for x, y in roman]
-            return pynini.string_map(roman_numerals)
-
-        cardinal_graph = CardinalFst(deterministic=True).graph
-        digit_teen = _load_roman("data/roman/digit_teen.tsv") @ cardinal_graph
-        ties = _load_roman("data/roman/ties.tsv") @ cardinal_graph
-        hundreds = _load_roman("data/roman/hundreds.tsv") @ cardinal_graph
+        roman_dict = load_labels(get_abs_path("data/roman/roman_to_spoken.tsv"))
+        default_graph = pynini.string_map(roman_dict).optimize()
+        default_graph = pynutil.insert("integer: \"") + default_graph + pynutil.insert("\"")
+        graph_teens = pynini.string_map([x[0] for x in roman_dict[:19]]).optimize()
```

Comment:
@ekmb shall we relax this too for audio based without lm?

Reply:
are there any confusing cases like abbreviations?